src/test/java/gov/healthit/chpl/aqa/stepDefinitions/CMSIDsLookupSteps.java

```diff
@@ -56,43 +64,39 @@ public void i_should_see_the_listings_that_make_up_the_CMS_ID(final String chplI
     
     @Given("^I am an end user on CHPL home page$")
     public void i_am_an_end_user() throws Throwable {
-        driver.get("https://chpl.ahrqstg.org/#/search");
-        WebDriverWait wait = new WebDriverWait(driver, DELAY);
-        driver.manage().window().maximize();
+        driver.get(url + "#/search");
+        WebDriverWait wait = new WebDriverWait(driver, TIMEOUT);
```

Comment:
This wait doesn't do anything, right? I mean, it won't actually wait until you do some sort of "wait.until()" call